app/views/contacts/_form.html.erb

```diff
@@ -40,13 +40,23 @@
     <%= f.text_field :availability %>
   </div>
   <div class="field">
-    <%= f.label :skillset %> 1 for junior, 2 for mid-level, 3 for expert 4 for overconfident! <br>
-    <%= f.number_field :skillset %>
+    <%= f.label :skillset %> <br>
+    <%= f.select :skillset, options_for_select(Contact.skillsets.keys, @contact.skillset) %>
```

Comment:
Definitely should add classes from Bootstrap for these things and form-group containers to space things out.

Not urgent. Nor required.